lib/hariko/resource/parser.js

```diff
@@ -98,6 +96,15 @@ exports.parse = function (rawdata) {
               entry.response.statusCode);
           }
         }
+
+        if (initialEntry != null) {
```

Comment:
__[jshint]__
* Use '!==' to compare with 'null'.

<!-- Comment by [SideCI](https://sideci.com) -->